app/scripts/lib/assertion.js

```diff
@@ -24,14 +24,13 @@ function (P, jwcrypto) {
     // browser has native crypto, no need to fetch entropy from the server.
     try {
       if (window.crypto && window.crypto.getRandomValues) {
-        return P(true); /*jshint ignore:line*/
+        return P(true);
       }
     } catch(e) {
       // some browsers blow up when trying to query window.crypto.
     }
 
     // wah wah, we need to get entropy from the server.
-    //jshint validthis: true
```

Comment:
@TDA I don't think so. I can't find that rule in the [CHANGELOG](https://github.com/eslint/eslint/blob/master/CHANGELOG.md) or on the [RULES](http://eslint.org/docs/rules/) page, only in that https://github.com/mozilla/fxa-content-server/pull/2597#discussion_r32895193 above.